install/windows/readme.md

```diff
@@ -3,16 +3,17 @@
 Sadly [binskim](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/) is only available on windows. So even for static analysis, a windows VM is required.
 
 ##Caution
-Use and separate Windows-VM for MobSF and *don't* expose it to a network range, where an attack might be coming from. The best solution is to set it to host-only mode.
+Use and separate Windows-VM for MobSF and *don't* expose it to a network range where an attack might be coming from. The best solution is to set it to host-only mode.
 
 ##Steps on the Windows-VM
 1. Install the following requirements on the VM
   * [Python 3](https://www.python.org/downloads/)
-  * Flask (via `python -m pip install flask`)
+  * Flask and rsa (via `python -m pip install flask rsa`)
```

Comment:
if the analysis is done on Host Windows. flask and rsa needs to go into `requirements.txt`